README.md

```diff
@@ -15,7 +15,7 @@ An apb for deploying [mariadb](https://www.mariadb.org/about/).
 * MARIADB_USER, default 'admin', mariadb database username.
 * OPENSHIFT_TARGET: Required, target openshift deployment
 * OPENSHIFT_TOKEN: Required, openshift token to authenticate with
-* MARIADB_VERSION, Optional, default '10.0', MariaDB version. 10.0 and 10.1 are supported.
+* MARIADB_VERSION, Optional, default '10.2', MariaDB version. 10.0, 10.1, and 10.2 are supported.
 
 ## Running the application
 `docker run -e "OPENSHIFT_TARGET=<openshift_target>" -e "OPENSHIFT_TOKEN=<token>" ansibleplaybookbundle/mariadb-apb provision`
```

Comment:
One change, this should be like the `asb-installer` instructions:

`docker run --rm --net=host -v $HOME/.kube:/opt/apb/.kube:z -u $UID docker.io/ansibleplaybookbundle/postresql-apb provision`